src/test/java/eu/adlogix/testutils/file/AdlTestFileUtils.java

```diff
@@ -0,0 +1,34 @@
+package eu.adlogix.testutils.file;
+
```

Comment:
This is a copy-paste code from the java test utils project. I didn't change the package name and kept the original package name, having moved this to src/test/java folder. Shall I keep it this way or move to an appnexus package?